java-lib/src/main/java/org/clink/linalg/ClinkSparseVector.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 The Clink Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.clink.linalg;
+
+import org.apache.flink.ml.linalg.SparseVector;
+
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+import com.sun.jna.Structure.FieldOrder;
+
+/** Class that corresponds to struct SparseVector in C++. */
+@FieldOrder({"n", "indices", "values", "length"})
+public class ClinkSparseVector extends Structure {
```

Comment:
Would it be better to rename this class to `SparseVectorJNA` to be more consistent with the C++ `SparseVectorJNA`? This could also make it clear that this class is only needed for conversion.

Reply:
OK. I'll make the change.